data/eos-updater-flatpak-installer.service.in

```diff
@@ -13,6 +13,29 @@ Type=oneshot
 RemainAfterExit=true
 ExecStart=@libexecdir@/eos-updater-flatpak-installer
 
+# Sandboxing
+# FIXME: Enable more of these options once we have systemd > 229
+Environment=GIO_USE_VFS=local
+Environment=GVFS_DISABLE_FUSE=1
+Environment=GVFS_REMOTE_VOLUME_MONITOR_IGNORE=1
+Environment=GSETTINGS_BACKEND=memory
+MemoryDenyWriteExecute=yes
+NoNewPrivileges=yes
+PrivateDevices=yes
+PrivateNetwork=no
+PrivateTmp=yes
+PrivateUsers=yes
+ProtectControlGroups=yes
+ProtectHome=yes
+ProtectKernelModules=yes
+ProtectKernelTunables=yes
+ProtectSystem=strict
```

Comment:
`ProtectSystem=strict` is overwritten by `ProtectSystem=no` immediately afterwards. Drop the `strict` one.